roles/oem/tasks/vm_only_post.yml

```diff
@@ -15,7 +15,7 @@
 # The USB 2/3 controllers are only available with the Virtualbox
 # Extensions which are non-free software and available only under the Oracle
 # PUEL. We should ensure we have not accidentally included them.
-- name: Validate missing USB 2/3 controllers
+- name: Validate missing USB 2/3 controllers # noqa 306
```

Comment:
Can we document why we don't use pipefail (or at least say that we plan to re-evaluate later)?